src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,60 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_CAPACITY = 10;
+    private final Pair<K,V>[]storage = (Pair<K,V>[]) new Pair[MAX_CAPACITY];
+    private int size = 0;
+
+    public static class Pair<K,V> {
```

Comment:
```suggestion
    private static class Pair<K,V> {
```
move it after public methods